spec/lines_classifier_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "helper"
+require "simplecov/lines_classifier"
+
+describe SimpleCov::LinesClassifier do
+  describe "#classify" do
+    describe "only relevant lines" do
+      it "classifies each line as relevant" do
+        classified_lines = subject.classify [
+          "def foo",
+          "end",
+        ]
+
+        expect(classified_lines.length).to eq 2
+        expect(classified_lines).to all be_relevant
```

Comment:
What is this magic I've never seen before :flushed: Cool stuff, thanks for teaching me!